Explorer/Assets/DCL/MapRenderer/MapPath/PinMarkerHelper.cs

```diff
@@ -11,7 +11,7 @@ namespace DCL.MapRenderer.MapLayers.Pins
 {
     public static class PinMarkerHelper
     {
-        public static async UniTaskVoid PulseScaleAsync(Transform transform, float scaleFactor = 1.5f, float duration = 0.5f, float delay = 0.1f, CancellationToken ct = default)
+        public static async UniTaskVoid PulseScaleAsync(Transform transform, float scaleFactor = 1.3f, float duration = 0.5f, float delay = 0.1f, CancellationToken ct = default)
```

Comment:
my bad, I didn't create some constants for the values, can you transform the magic numbers into constants?

Reply:
yep yep

Reply:
Also, actually those are my bads xDD